lib/templates/sharedPart.js

```diff
@@ -97,11 +97,7 @@ class SharedPart {
 
   static async #findTemplateHandle(firmId, template) {
     // Search in repository
-    let handle = await fsUtils.findHandleByID(
-      firmId,
-      template.type,
-      template.id
-    );
+    let handle = fsUtils.findHandleByID(firmId, template.type, template.id);
```

Comment:
@Woetfin any specific reason to remove the `await` ?

Reply:
VS Code let me auto-remove it because they said it's unnecessary, so I followed that advice